scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -164,6 +164,68 @@ function doStartEclipse() {
   fi
 }
 
+function doStartEclipseBlocking() {
+  if doIsWindows
+  then
+    # shellcheck disable=SC2086
+    "eclipse" /B "${ECLIPSE_HOME}/eclipsec" -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${JAVA_HOME}/bin/javaw" -showlocation "${WORKSPACE}" -vmargs ${ECLIPSE_VMARGS}
+  else
+    # shellcheck disable=SC2086
+    "${ECLIPSE_HOME}/eclipse" -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${JAVA_HOME}/bin/java" -showlocation "${DEVON_IDE_HOME/*\//}/${WORKSPACE}" -vmargs ${ECLIPSE_VMARGS}
+  fi
+}
+
```

Comment:
Sorry, did not see the code when I commented this. Forget my comment :)

Reply:
@hohwille an unlucky coincidence in formulation. I know he was talking about a "conditional block" that determines what OS is being used. But my solution also involves running a process from the shell without using `start`/`$` (which is called "backgrounding" instead of "blocking" I now realize).

My inital plan was to detect, when the launched eclipse instance is closed so I can roll back the changes made to `workspace.setup` (this file must be restored to its inital state, otherwise Eclipse will run a time intensive import task on every startup).

Since the duration of Eclipse import tasks varies in duration depending on the size/complexity of the project and the system hardware, I see no other way to determine when `workspace.setup` can be reset other than waiting for that Eclipse instance to terminate.

By not launching Eclipse with `start`/`$` I wanted to make sure that the user closes the Eclipse instance, before my shell script continues and rolls back the file changes.